package.json

```diff
@@ -151,12 +151,12 @@
     "typescript-eslint": "^8.39.0",
     "unplugin": "^1.7.1",
     "uuid": "^9.0.0",
-    "vite": "^5.4.7",
+    "vite": "^7.1.1",
     "vite-plugin-devtools-json": "^0.4.1",
-    "vite-plugin-glslify": "^2.2.1",
-    "vite-plugin-inspect": "0.8.9",
-    "vite-plugin-top-level-await": "^1.5.0",
-    "vite-plugin-wasm": "^3.4.1",
+    "vite-plugin-glslify": "^2.3.0",
+    "vite-plugin-inspect": "11.3.2",
```

Comment:
probably don't need this pinned anymore, it was before because we were on an old version and couldn't upgrade until we upgraded vite